TypedSyntax/test/runtests.jl

```diff
@@ -1,9 +1,11 @@
 using JuliaSyntax: JuliaSyntax, SyntaxNode, children, child, sourcetext, kind, @K_str
 using TypedSyntax: TypedSyntax, TypedSyntaxNode
 using Dates, InteractiveUtils, Test
+using Core.Compiler: widenconst
 
-has_name_typ(node, name::Symbol, @nospecialize(Ts::Tuple)) = kind(node) == K"Identifier" && node.val === name && node.typ in Ts
-has_name_typ(node, name::Symbol, @nospecialize(T)) = kind(node) == K"Identifier" && node.val === name && node.typ === T
+has_typ(node, @nospecialize(Ts::Tuple)) = any(has_typ(node, T) for T in Ts)
```

Comment:
Just a minor nit: It would be usually better to avoid iterating over tuple whose type isn't statically known, since it can result in less profitable code generation.
```suggestion
has_typ(node, @nospecialize(Ts::Tuple)) = any(has_typ(node, Ts[i]) for i in 1:length(Ts))
```